omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java

```diff
@@ -45,6 +50,14 @@
 	protected void initialize() {
 		// ensure properties for this class are properly injected
 		// TODO find a way around this hack!
+		int defaultPageSize = Integer.parseInt(globalPropertyService.getGlobalProperty(FhirConstants.OPENMRS_FHIR_DEFAULT_PAGE_SIZE));
```

Comment:
We should use [`NumberUtils#toInt()`](https://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/math/NumberUtils.html#toInt-java.lang.String-int-) to parse this.